pkg/sti/api/script.go

```diff
@@ -0,0 +1,20 @@
+package api
+
+// Script defines script names used by STI process.
+type Script string
```

Comment:
This was done to remove all those strings from the code, of course I could switch all the string into constants, but this is more readable IMHO.